felix/bpf/tc/attach.go

```diff
@@ -406,8 +373,8 @@ func (ap *AttachPoint) Config() string {
 	return fmt.Sprintf("%+v", ap)
 }
 
-func (ap *AttachPoint) ConfigureProgram(m *libbpf.Map) error {
-	globalData := libbpf.TcGlobalData{
+func (ap *AttachPoint) ConfigureProgram() *libbpf.TcGlobalData {
```

Comment:
```suggestion
func (ap *AttachPoint) Configuration() *libbpf.TcGlobalData {
```

It would make sense to name it like this to allign with XDP, also it just returns configuration